### ISSUE TITLE
Depositing DNA Problems

### ISSUE THREAD

**Issue by willspeed37** [mvs-org/lightwallet#341]
I have tried depositing DNA from 2 different  exchanges to my etp lightwallet  and both times they require a memo and there isn't one to give them?  I deposited  ETP to my wallet and it was a piece of cake  although ETP dosent require a memo... Please help i have a lot of DNA sitting on exchanges with no way to withdraw them?   And if possible could you tell me if there is any other wallet i can use to store my DNA crypto or is there only the MY ETP Light wallet?    Thank you.

**Comment by aaronzhzh** [comment]
The reason why memo is needed is because DNA has been migrated into DNA mainnet token, you need to visit https://mvsdna.info, after creating a DNA wallet, withdraw cash from the exchange (please remember to back up the seed phrase).

**Comment by willspeed37** [comment]
you guys are funny this is the 2nd response you"ve sent to my support ticket and both of your responses are useless.... I will say it again, i have DNA on 2 different exchanges and i cannot withdraw it without a memo! You do not provide a memo for my wallet on the mainnet just an address. That's the problem (telling me why a memo is needed does not help me at all) I need you to provide me a memo that goes with the deposit address? otherwise i cannot with draw my DNA crypto. I might as well sell it and put the proceeds into another crypto...... Most crypto's don't require a memo/tag to withdraw or deposit i can only think of a couple off the top of my head.. hbar & xrp do... By the way both exchanges support dept. told me they can't help me That's why i contacted you......

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by aaronzhzh** [comment]
Have you ever tried to submit a withdrawal request without memo on the exchange? Maybe memo is not required.

**Comment by aaronzhzh** [comment]
Just fill in your DNA receiving account , without memo,try to withdraw a small amount.

**Comment by willspeed37** [comment]
yes i have tried that.. it won't let me withdraw without it. MXC is 1 of the exchanges and Bittrex is the other..

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by aaronzhzh** [comment]
![b319d44c4dd333be32afe83f91874bd](https://user-images.githubusercontent.com/61041995/117600206-18807700-b17e-11eb-98c5-6b6db3bd1875.jpg)
Click “No Memo”,then sitll not working?

**Comment by willspeed37** [comment]
Looks like it would work doesn't it? If you just put in your withdraw address it won't let complete the withdraw but if you put in your avatar name which i did after i created it and paid the 1 ETP crypto.. It goes through like the withdraw works, But NO..... It's a fake out It sits there saying it's processing but never goes through and then the MXC people cancel the withdraw. After i tried that a couple of times and they kept canceling it, I contacted there support dept. and the support guy tried it again and it didn't work and he finally said he would send it over to the tech team whatever that is and i never heard back from them that was a week ago or so and the other exchange flat out told me they couldn't help me and told me to reach out to you. So here we are now you know how much this crap blows! Well good luck trying to fix this... Unless im the only one on the whole planet that is trying to withdraw DNA crypto i don't see how you have not ran into this problem before?
Hopefully you can fix it....Thank you Mr Chung for getting back to me in a timely fashion.

Sent with [ProtonMail](https://protonmail.com) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by aaronzhzh** [comment]
First of all, you need to make sure that your withdrawal account is a DNA mainnet account, not an address starting with M on the ETP-DNA mainnet. DNA token has been migrated to the DNA mainnet. Currently MXC can only withdraw new DNA tokens, so you need to create a DNA wallet (not lightwallet) at https://mvsdna.info/, and then enter your DNA account on the MXC page (and Not the avatar of etp mainnet).
**##Please make sure that the account you withdraw is the DNA mainnet account, not the address or avatar of lightwallet##**

**Comment by willspeed37** [comment]
I’ll try that…thanks for your help.

**Comment by aaronzhzh** [comment]
ur welcome. if u need more help pls comment below. i will reply you asap

**Comment by willspeed37** [comment]
Ok..Thanks again
